backend/src/controllers/auth/logout.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from 'express';
+
+import { TMiddlewareCall } from 'interfaces/commonMiddleware';
+import { isError } from 'utils/typeGuards/isError';
+import { verifyAccessToken } from 'utils/auth/authUtils';
+import { clearTokenProvider } from 'db/providers/authProvider';
+
+const logout = async (req: Request, res: Response, next: TMiddlewareCall) => {
+  try {
+    const accessToken = req.headers.authorization?.split(' ')[1];
```

Comment:
I would add an empty line here to separate logic of getting token value and verifying it.